ospf6d/ospf6_interface.c

```diff
@@ -435,6 +434,35 @@ void ospf6_interface_connected_route_update(struct interface *ifp)
 			}
 		}
 
+		if (oi->state == OSPF6_INTERFACE_LOOPBACK
```

Comment:
might want to look at #8061 here just for coordination

Reply:
I looked at that before working on this & it looked like it was inactive (and it's autoclose now.)  Either way the changes in this PR include a superset of #8061 (since the LA bit has additional rules for PtMP support.)